### PR TITLE
Add Trophy

### PR DESCRIPTION
**Pull request by yuweol**
Including aosp, tizen, and other famous opensource projects.

### CODE REVIEW

**Review by sole2 — APPROVED** [review]
LGTM :+1:

**Review by musangk — DISMISSED** [review]
LGTM 👍

File: Trophy.md [hunk 1]
```diff
@@ -56,6 +56,35 @@ Project | Bugs or CVEs | Category | Fuzzer
...
+[tizen/privilege-info](https://review.tizen.org/gerrit/#/admin/projects/platform/core/security/privilege-info) | [#255010](https://review.tizen.org/gerrit/#/c/platform/core/security/privilege-info/+/255010/) | `nullchk` |
+[tizen/libmm-fileinfo](https://review.tizen.org/gerrit/#/admin/projects/platform/core/multimedia/libmm-fileinfo) | [#255012](https://review.tizen.org/gerrit/#/c/platform/core/multimedia/libmm-fileinfo/+/255012/) | `bof` |
+[tizen/alarm-manager](https://review.tizen.org/gerrit/#/admin/projects/platform/core/appfw/alarm-manager) | [#255345](https://review.tizen.org/gerrit/#/c/platform/core/appfw/alarm-manager/+/255345/) | `nullchk` |
+[tizen/context](https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/context) | [#255046](https://review.tizen.org/gerrit/#/c/platform/core/api/context/+/255046/) | `other` |
+[tizen/context](https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/context) | [#255046(2)](https://review.tizen.org/gerrit/#/c/platform/core/api/context/+/255046/) | `other` |
```
Comment (sole2) [inline Trophy.md]: There is duplicate row.
```suggestion
[tizen/context](https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/context) | [#255046](https://review.tizen.org/gerrit/#/c/platform/core/api/context/+/255046/) | `other` |
```

File: Trophy.md [hunk 2]
```diff
@@ -56,6 +56,35 @@ Project | Bugs or CVEs | Category | Fuzzer
...
+[tizen/context](https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/context) | [#256732](https://review.tizen.org/gerrit/#/c/platform/core/api/context/+/256732/) | `free` |
+[tizen/mediatool](https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/mediatool) | [#257660](https://review.tizen.org/gerrit/#/c/platform/core/api/mediatool/+/257660/) | `bof` |
+[tizen/pkgmgr-info](https://review.tizen.org/gerrit/#/admin/projects/platform/core/appfw/pkgmgr-info) | [#257518](https://review.tizen.org/gerrit/#/c/platform/core/appfw/pkgmgr-info/+/257518/) | `nullchk` |
+[tizen/system-settings](https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/system-settings) | [#258170](https://review.tizen.org/gerrit/#/c/platform/core/api/system-settings/+/258170/) | `nullchk` |
+[tizen/system-settings](https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/system-settings) | [#258170(2)](https://review.tizen.org/gerrit/#/c/platform/core/api/system-settings/+/258170/) | `nullchk` |
```
Comment (sole2) [inline Trophy.md]: There is duplicate row.
```suggestion
[tizen/system-settings](https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/system-settings) | [#258170](https://review.tizen.org/gerrit/#/c/platform/core/api/system-settings/+/258170/) | `nullchk` |
```

**Review by sole2 — CHANGES_REQUESTED** [review]
Found duplicate rows.

File: Trophy.md (hunk 2 above)
Comment (yuweol) [reply Trophy.md]: Fixed 7d9f540

File: Trophy.md (hunk 1 above)
Comment (yuweol) [reply Trophy.md]: Fixed 7d9f540

**Review by sole2 — APPROVED** [review]
LGTM :+1:

**Review by musangk — APPROVED** [review]

**Pull request merged**